src/main/java/org/folio/rest/support/StatusUpdatedDateGenerator.java

```diff
@@ -1,11 +1,12 @@
 package org.folio.rest.support;
 
-import java.text.SimpleDateFormat;
-import java.util.Date;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
 
 public final class StatusUpdatedDateGenerator {
+    private final static DateTimeFormatter df = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
     public static String generateStatusUpdatedDate() {
-      SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
-      return sdf.format(new Date());
-    } 
+      return df.format(ZonedDateTime.now(ZoneOffset.UTC));
```

Comment:
@marcjohnson-kint I had to add the UTC offset due to issues I was seeing in another module. I ran some tests prior to verify that the dates generated by the most recent change was the same as the previous version. With this change, I am no longer leaving the timezone to change.

Reply:
> @marcjohnson-kint I had to add the UTC offset due to issues I was seeing in another module. I ran some tests prior to verify that the dates generated by the most recent change was the same as the previous version. 

Ok. Whilst I think it _should_ be unnecessary to do this, it is the polite thing to do (and some parts of FOLIO expect it)

> With this change, I am no longer leaving the timezone to change.

Do you mean chance?